docs/_sources/install-riallto-windows.rst

```diff
@@ -38,7 +38,7 @@ Download Riallto installer
 
 Riallto consists of runtime software to load Ryzen AI applications, and a software toolchain to compile and build applications for Ryzen AI. Riallto also includes a series of Jupyter Notebook tutorials. 
 
-`Download the latest v1.0 Riallto installer <https://www.xilinx.com/bin/public/openDownload?filename=Riallto-v1.0.zip>`_ and run it on your Ryzen AI laptop. This will install the Riallto software framework and a copy of the Riallto Jupyter notebooks that you can browse and run on your laptop. Make sure to select the correct installation option when prompted, the 'Full' version will not install without WSL2 or the AIE build license (see details in the Prerequisites section above). 
+`Download the latest v1.0 Riallto installer <https://www.xilinx.com/bin/public/openDownload?filename=Riallto-v1.1.zip>`_ and run it on your Ryzen AI laptop. This will install the Riallto software framework and a copy of the Riallto Jupyter notebooks that you can browse and run on your laptop. Make sure to select the correct installation option when prompted, the 'Full' version will not install without WSL2 or the AIE build license (see details in the Prerequisites section above). 
```

Comment:
We should change url to:
https://www.amd.com/bin/public/amdOpenDownload?filename=Riallto-v1.1.zip

Though https://www.xilinx.com/bin/public/openDownload?filename=Riallto-v1.1.zip still works because I assume they're mirrored

Reply:
Updated.